app/views/expenses/new.html.erb

```diff
@@ -0,0 +1,54 @@
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Expense Transactions app</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
```

Comment:
- [ ] Please implement the back button as requested by the previous reviewer. 🙏🏼

Reply:
- [ ] Please implement the back button as requested by the previous reviewer.